apps/buckler_bot/lib/buckler_bot/handler.ex

```diff
@@ -114,15 +95,47 @@ defmodule BucklerBot.Handler do
       }
     }
   }, _) do
-    case Repo.user_unauthorized?(chat_id, user_id) do
+    Logger.debug("New media message!")
+    case Connections.user_unauthorized?(chat_id, user_id) do
       {true, user} ->
-        Repo.delete_user(chat_id, user_id)
+        process_captcha_check(false, conn, message_id, user)
+      _ -> conn |> Agala.Conn.halt
+    end
+  end
+
+  def process_captcha_check(true, conn, message_id, user) do
+    multi do
+      add delete_message(conn, user.chat_id, user.welcome_message_id)
+      add delete_message(conn, user.chat_id, message_id)
+      Connections.delete_chatuser(user.chat_id, user.user_id)
```

Comment:
Maybe after successful authorization need to send message with some welcome text?
p/s/ we can create issue for this

Reply:
No, user will understand they everything is alright because he's not banned. Others don't need to see any mess

Reply:
This is to assist in debugging, because now it is not clear on history of chat what happened - ban or authorization...
Maybe configure this behaviour through `config.exs`?